docs/data_pack.md

```diff
@@ -1,5 +1,31 @@
 # DataPack Tutorial
 
+## Get primitive data
+`DataPack.get_data()` is commonly used to retrieve primitive data from a `DataPack`. User can request particular data fields within the range of a particular `Annotation` or `AudioAnnotation` type. User request particular data fields by setting `request` and the search range by setting `context_type`.
```

Comment:
Check the grammar with some tools. For example, you can say "A user", "The user" or "Users".